tracing-fmt/Cargo.toml

```diff
@@ -1,8 +1,9 @@
 [package]
 name = "tracing-fmt"
-version = "0.1.0"
+version = "0.0.1-alpha.1"
 authors = ["Eliza Weisman <eliza@buoyant.io>"]
 edition = "2018"
+license = "MIT"
 
```

Comment:
Can we add more cargo metadata here? It can mostly be similar to `tracing`'s:

https://github.com/tokio-rs/tracing/blob/0a5ba16c5fea076f4902995bd59dedd99cac2755/tracing/Cargo.toml#L15-L27

We should also add 
```toml
[badges] 
maintenance = { status = "experimental" }
```
until we publish an 0.1